stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -1792,13 +1788,18 @@ pub mod test {
         nonce: u64,
         addr: PoxAddress,
         lock_period: u128,
+        signer_key: Vec<u8>,
```

Comment:
If we want to be extra strict, this should actually be a [u8; 33] (or a Point), but not a deal breaker.

Reply:
I wasn't sure how strict we should be. `Point` is fine by me if you agree.